content/en/apps/reference/app-settings/outbound.md

```diff
@@ -7,7 +7,7 @@ description: >
 relevantLinks: >
 ---
 
-*Outbound is only available in Medic 3.5.0 and above*
+*Outbound is only available in cht-core 3.5.0 and above*
```

Comment:
In most other places we mention versions we don't specify cht-core (or medic), but rather just say "available in X". The extra clarity is helpful though, and for consistency should use "CHT Core" rather than the "cht-core".

```suggestion
*Outbound is only available in CHT Core 3.5.0 and above*
```

Reply:
Awesome - thanks for the style guide help there @abbyad  - much obliged.